tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -15,7 +15,7 @@
 AZURE_TENANT_ID = os.environ.get("AZURE_TENANT_ID")
 worker_count = int(os.environ.get('RANCHER_STRESS_TEST_WORKER_COUNT', 1))
 HOST_NAME = os.environ.get('RANCHER_HOST_NAME', "testcustom")
-engine_install_url = "https://releases.rancher.com/install-docker/19.03.sh"
+engine_install_url = "https://releases.rancher.com/install-docker/20.10.sh"
```

Comment:
Even though we dont usually run automation from this branch, we should not be changing values in files under tests/validation as we dont always run automation tests on 2.5.6+